node/src/types/node_id.rs

```diff
@@ -46,7 +46,7 @@ enum NodeIdAsString {
 impl Serialize for NodeId {
     fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
         if serializer.is_human_readable() {
-            NodeIdAsString::Tls(hex::encode(&self.0)).serialize(serializer)
+            NodeIdAsString::Tls(base16::encode_lower(&self.0)).serialize(serializer)
```

Comment:
```suggestion
            NodeIdAsString::Tls(checksummed_hex::encode(&self.0)).serialize(serializer)
```

Reply:
My comment on the `deserialize()` method was meant for here - but same story - I'll omit this suggestion.